### PR TITLE
feat: add mobile menu

### PR DESCRIPTION
**Pull request by denys-log**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -38,7 +38,16 @@ <h1>Example page title</h1>
...
+  <load ="partials/mobile-menu.html" />
+
   <script type="module" src="./main.js"></script>
+  <script
+    defer
+    src="https://cdnjs.cloudflare.com/ajax/libs/body-scroll-lock/3.1.5/bodyScrollLock.min.js"
+    integrity="sha512-HowizSDbQl7UPEAsPnvJHlQsnBmU2YMrv7KkTBulTLEGz9chfBoWYyZJL+MUO6p/yBuuMO/8jI7O29YRZ2uBlA=="
+    crossorigin="anonymous"
+  ></script>
+  <script type="module" src="./js/mobile-menu.js"></script>
```
Comment (kharkov-timur) [inline src/index.html]: mobile-menu.js should be imported inside main.js file
Please fix it.

File: src/css/mobile-menu.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  background: transparent;
+  padding: 0;
+  margin: 0;
+  border: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.mobile-menu-close button,
```
Comment (kharkov-timur) [inline src/css/mobile-menu.css:72]: Please add hover effect for button

**Review by kharkov-timur — CHANGES_REQUESTED** [review]

**Comment by Fijji** [comment]
Please, do the cleanup > remove old structure file "menu_mob.html"

File: src/css/mobile-menu.css (hunk 2 above)
Comment (denys-log) [reply src/css/mobile-menu.css:72]: done

File: src/index.html (hunk 1 above)
Comment (denys-log) [reply src/index.html]: done

**Review by kharkov-timur — APPROVED** [review]

**Pull request merged**